packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -64,6 +64,9 @@ class Checkbox extends StatelessWidget {
   /// If null, the checkbox will be displayed as disabled.
   final ValueChanged<bool> onChanged;
 
+  /// The width of a checkbox widget.
+  static const double width = 18.0;
```

Comment:
Yeah, that seems like a good idea. File a bug maybe? Or we can just wait til we need it.